test/suite/helper_tiered.py

```diff
@@ -27,6 +27,11 @@
 # OTHER DEALINGS IN THE SOFTWARE.
 #
 
+try:
```

Comment:
I'm not sure if this is the best way to import the library but we need to account for the test runs that don't have boto3 installed.